test/system/600-completion.bats

```diff
@@ -316,3 +335,40 @@ function _check_completion_end() {
     done <<<"$output"
 
 }
+
+@test "podman shell completion for paths in container/image" {
+    for cmd in create run; do
+        run_completion $cmd $IMAGE ""
+        assert "$output" =~ ".*^/etc\$.*^/home\$.*^/root\$.*" "root directories suggested (cmd: podman $cmd)"
+
+        # check completion for subdirectory
+        run_completion $cmd $IMAGE "/etc"
+        # It should be safe to assume the os-release file always exists in $IMAGE
+        assert "$output" =~ ".*^/etc/os-release\$.*" "/etc files suggested (cmd: podman $cmd)"
+        # check completion for partial file name
+        run_completion $cmd $IMAGE "/etc/os-"
+        assert "$output" =~ ".*^/etc/os-release\$.*" "/etc files suggested (cmd: podman $cmd)"
```

Comment:
Suggestion: `podman $cmd /etc/os-`

(this disambiguates each specific condition, and makes it much easier for the end user to identify the problem)